POLICY.txt

```diff
@@ -44,14 +49,20 @@ Please make sure that you include the following information:
 - Which service is affected
 - How can the bug be used/exploited
 - Explanation of the risk
+- If possible, include a estimated CVSS score
 
 Reports will be answered within 48 hours. If you have not received an
-answer within that time frame, feel free to contact us again.
+answer within that time frame, feel free to contact us again. Please do
+not ask for updates on a ticket repeatedly as it may take time to
+resolve the issue.
 
 For used open source software, we recommend to file bug reports and/or
 pull requests against the upstream repositories. This includes hardening
 instructions in the installation documentation.
 
+If you are reporting a known vulnerability, please include a reference
+to the original vulnerability report.
```

Comment:
... and/or a CVE number.

Please stick to the methods defined out in the Common Vulnerability Scoring System (CVSS) for defining the criticality of a vulnerability.